_config.yml

```diff
@@ -11,7 +11,12 @@ markdown: kramdown
 
 # Custom Variables
 
+main:
+    plugins: "assets/plugins"
+    css: "assets/css"
+    js: "assets/js"
+
 path:
```

Comment:
The /about/ is because that is what is specified in the front matter of the markdown file. 
https://github.com/secseattle/secseattle.github.io/pull/17/files#diff-6083a88ee3411b0d17ce02d738f69d47R4